### ISSUE TITLE
[Bug: Release 2.0]:  executing "partials/footer.html" at <js>: can't evaluate field Build in type string

### ISSUE THREAD

**Issue by sophiabrandt** [panr/hugo-theme-terminal#180]
Hello,
thanks for the continued work on the theme!

I've tried updating my theme today, but keep getting errors.

I've created a minimal (fresh) repo that reproduces the problem here: [minimal failing Hugo](https://github.com/sophiabrandt/theme-terminal-release-2).

Hugo version: 0.73.0/extended linux/amd64

Errors:
```sh
Building sites … ERROR 2020/08/10 17:15:18 render of "taxonomy" failed: execute of template failed: template: _default/terms.html:22:7: executing "footer" at <partial "footer.html" .>: error calling partial: "/home/user/Coding/Go/theme-terminal-release-2/themes/terminal/layouts/partials/footer.html:15:39": execute of template failed: template: partials/footer.html:15:39: executing "partials/footer.html" at <js>: can't evaluate field Build in type string
ERROR 2020/08/10 17:15:18 render of "taxonomy" failed: execute of template failed: template: _default/terms.html:22:7: executing "footer" at <partial "footer.html" .>: error calling partial: "/home/user/Coding/Go/theme-terminal-release-2/themes/terminal/layouts/partials/footer.html:15:39": execute of template failed: template: partials/footer.html:15:39: executing "partials/footer.html" at <js>: can't evaluate field Build in type string
ERROR 2020/08/10 17:15:18 failed to render pages: render of "home" failed: execute of template failed: template: _default/index.html:22:7: executing "footer" at <partial "footer.html" .>: error calling partial: "/home/user/Coding/Go/theme-terminal-release-2/themes/terminal/layouts/partials/footer.html:15:39": execute of template failed: template: partials/footer.html:15:39: executing "partials/footer.html" at <js>: can't evaluate field Build in type string
Built in 6 ms
Error: Error building site: POSTCSS: failed to transform "css/style.css" (text/css): resource "css/css/style.css_23cbd773103b4c27602bf83ce1d529de" not found in file cache
```

It's probably an error with the Hugo pipelines/NPM package installation.
I've tried both installing the packages globally and locally _from the main Hugo folder_.

Any help is appreciated!

**Comment by panr** [comment]
Hi sophiabrandt! You get the error because of the Hugo version. In v2 I used the latest pipes functions which are available from 0.74.x.

**Comment by sophiabrandt** [comment]
Aww, dang!
I must have overlooked that. I will wait for an update of Hugo for my OS then. Thanks!
